stl/inc/yvals_core.h

```diff
@@ -1195,7 +1197,10 @@
 #define _DEPRECATE_NONFLOATING_COMPLEX
 #endif // ^^^ warning disabled ^^^
 
-// next warning number: STL4038
+// STL4038 is used to warn about the contents of some header is available only with specific C++ standard or later. See
+// example at stl/inc/any, line 13

```

Comment:
The first sentence has minor grammar issues, which could be repaired by just mentioning a generalized example.

The second sentence is prone to rotting (any added/removed lines at the top will invalidate the line number); I think it can be omitted as "STL4038" is highly searchable, just a Ctrl+Shift+F away.